cmd/wallet-web/src/pages/Signup.vue

```diff
@@ -135,16 +137,23 @@ export default {
   data() {
     return {
       providers: [],
-      hubOauthProvider: this.hubURL(),
       statusMsg: '',
       loading: true,
       registered: false,
+      isLog: false,
```

Comment:
Can we more descriptive with the variable name ?

Reply:
This is unused now, Removed